packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { elementTypeAcceptingRef } from '@material-ui/utils';
 import { useThemeProps } from '@material-ui/system';
+import { NoSsr } from '@material-ui/unstyled';
```

Comment:
Since `@material-ui/unstyled` is meant to have a nonnegligible package size. I think that we can import deep to force three-shaking in dev mode.

```suggestion
import NoSsr from '@material-ui/unstyled/NoSsr';
```